api/pkg/apis/v1alpha1/providers/config/catalog/catalogprovider_test.go

```diff
@@ -107,7 +113,24 @@ func TestRead(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "name", summary[0].Name)
 
-	res, err = provider.Read(ctx, "catalog1:v1", "parentAttribute", nil)
+	res, err = provider.Read(ctx, "catalog1:v1", "`.a.b.c`", nil)
```

Comment:
Is .a.b.c considered a jq query? Shouldn't it be wrapped in single quotes?

Reply:
Yes, it is a jq query, it is quoted with ``, which will be recognized by jq as directly